ibm/config.go

```diff
@@ -154,6 +177,13 @@ func (c *Config) ClientSession() (interface{}, error) {
 		return session, nil
 	}
 
+	userConfig, err := fetchUserDetails(sess.BluemixSession)
+	if err != nil {
+		log.Printf("Error occured fetching the account id/user id %v", err.Error())
+	}
```

Comment:
I think you can set the value of bmxUserFetchErr here.

bmxUserFetchErr := fmt.Errorf(...)